index.js

```diff
@@ -24,7 +24,10 @@ exports.debug_mode = /\bredis\b/i.test(process.env.NODE_DEBUG);
 
 // hiredis might not be installed
 try {
-    parsers.push(require("./lib/parsers/hiredis"));
+    // Confuse bundlers so that they won't try
+    // to include this native module.
+    var hiredisParserPath = "./lib/parsers/hiredis";
+    parsers.push(require(hiredisParserPath));
```

Comment:
I suggest adding a comment like `// Native module` instead. That way you could parse the comments while bundling and do not rely on anything else. 
Is your bundling logic public?